httpserver/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 2.8.0)
 
 add_definitions(-DBOOST_LOG_DYN_LINK)
+add_definitions(-DDISABLE_ENCRYPTION)
```

Comment:
This is intentional, right?

Reply:
Yes, the more I think about it tho the more I think we won't be re-enabling the encryption tho